cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java

```diff
@@ -108,5 +176,18 @@ public String getFileName() {
             return fileName;
         }
     }
-}
 
+    /**
+     * Returns the name of the test file according to the type
+     * @return name of the test file
+     */
+    public String getTestFileName() {
+
+        if (SrcFileType.MAIN == getSrcFileType()) {
+            return "main_test.bal";
+        } else if (SrcFileType.SERVICE == getSrcFileType()) {
+            return "hello_service_test.bal";
+        }
+        return "test.bal";
```

Comment:
Shall we make testFileName a class variable and resolve the value in the constructor itself as the other values?